src/components/book/BookCard.js

```diff
@@ -15,7 +15,7 @@
 
 import React, { Component, Fragment } from "react";
 import { withRouter } from "react-router-dom";
-
+// import { Link } from "react-router-dom";
```

Comment:
Hello! Friendly reminder to delete unused comments ^^

Reply:
I will fix it.

Reply:
I removed unused comments in BookCard.js